player/lua/select.lua

```diff
@@ -62,6 +62,25 @@ mp.add_forced_key_binding(nil, "select-playlist", function ()
     })
 end)
 
+local function format_flags(track)
+    local flags = ""
+
+    for _, flag in ipairs({
+        "default", "forced", "dependent", "visual-impaired", "hearing-impaired",
+        "image", "external"
```

Comment:
Do we need both image and albumart?

Reply:
My idea is, if that's a separate flag in our code, they both should be here. I can imaging some can have one or the other flag, no?

Reply:
No, I made all demuxers set image to true for attached pictures. They are separate flags because albumart predates image, it probably wouldn't be worth exposing now.

Reply:
Also `albumart` is a misnomer for mkv thumbnails.

Reply:
Fine, if you say so.